connectors/gridgo-mysql/src/main/java/io/gridgo/connector/mysql/JdbcConnector.java

```diff
@@ -0,0 +1,37 @@
+package io.gridgo.connector.mysql;
+
+import io.gridgo.connector.impl.AbstractConnector;
+import io.gridgo.connector.support.annotations.ConnectorEndpoint;
+import io.gridgo.framework.support.exceptions.BeanNotFoundException;
+import org.jdbi.v3.core.ConnectionFactory;
+import snaq.db.ConnectionPool;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Optional;
+
+@ConnectorEndpoint(scheme = "jdbc", syntax = "jdbcUri", raw = true)
+public class JdbcConnector extends AbstractConnector {
+
+    private static HashSet<String> reserveParams = new HashSet<>(Arrays.asList("pool"));
+
+    protected void onInit() {
+        var userName = getParam("user");
+        var password = getParam("password");
+        var connectionBean = getParam("pool");
+        try {
+            var connectionFactory = getContext().getRegistry().lookupMandatory(connectionBean, ConnectionFactory.class);
```

Comment:
(!) should check if `connectionBean == null` instead of catching exception. if the users pass the `pool` parameter but the bean is not registered we should throw exception instead of using a default pool.